webpack/scenes/Content/ContentPage.js

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Grid, GridItem, TextContent, Text, TextVariants } from '@patternfly/react-core';
+import { isEmpty } from 'lodash';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { STATUS } from 'foremanReact/constants';
+import ContentTable from './Table/ContentTable';
+import { selectContentTypes, selectContentTypesStatus } from './ContentSelectors';
+import ContentConfig from './ContentConfig';
+import { getContentTypes } from './ContentActions';
+import Loading from '../../components/Loading';
+import EmptyStateMessage from '../../components/Table/EmptyStateMessage';
+
+const ContentPage = () => {
+  const dispatch = useDispatch();
+  const contentTypesResponse = useSelector(selectContentTypes);
+  const contentTypesStatus = useSelector(selectContentTypesStatus);
+  const [selectedContentType, setSelectedContentType] = useState(null);
+  const [contentTypes, setContentTypes] = useState(null);
+
+  useEffect(() => {
+    // Set list of enabled content types and initial content type
+    const buildContentTypes = () => {
```

Comment:
I moved `buildContentTypes` inside of the `useEffect` because it became a dependency after I changed the function syntax. Not sure if this is the best way to handle it.

Reply:
Ya..That's good..